packages/cosmos_ui_components/lib/components/cosmos_circle_text_button.dart

```diff
@@ -3,14 +3,16 @@ import 'package:flutter/material.dart';
 
 class CosmosCircleTextButton extends StatelessWidget {
   final String text;
+  final String asset;
   final VoidCallback? onTap;
   final IconData icon;
```

Comment:
you're not using `icon` anymore, lets remove it

Reply:
Removed